server/api/team/team.spec.js

```diff
@@ -24,12 +28,12 @@ describe('GET /api/teams/:id/players', function() {
 
   before(function(done) {
     // Clear db before testing
-    db.Team.destroy({},{truncate: true}).success(function() {
-      db.Player.destroy({}, {truncate: true}).success(function() {
-        db.PlayerAssignment.destroy({}, {truncate: true}).success(function() {
-        done();
-        });
-      });
+    db.Team.destroy({},{truncate: true}).then(function() {
+      return db.Player.destroy({}, {truncate: true})
```

Comment:
I accidentally updated our version of Sequelize so if you run 'npm install' post-merging you may have to call ".destroy({where: true})" rather than ".destroy({},{truncate: true})"